### DISCUSSION TITLE
Grub dual boot

### DISCUSSION THREAD

**Discussion by sudobrat** [prasanthrangan/hyprdots#932] [Q&A]
I have dual boot windows 11 and this OS.
I need the grub to show both os and I can select which one I want to go to. How can I do that please lemme know

Thanks in Advance 🙏

**Comment by RookiexCookie** [comment]
installed os-prober using this line `sudo pacman -S os-prober`
then after installing go edit this file `sudo nano /etc/default/grub`
and find this line `GRUB_DISABLE_OS_PROBER=false` it would be most likely commented so uncomment it then
run this `sudo grub-mkconfig -o /boot/grub/grub.cfg`
then reboot , the grub should show both os

**Comment by sudobrat** [comment]
alright! thanks a lot for your help :)
